assets/tests/components/mapPage.test.tsx

```diff
@@ -347,15 +381,46 @@ describe("<MapPage />", () => {
       tripId === vehicle.tripId ? shapes : null
     )
 
+    const { container } = render(<MapPage />, {
+      wrapper: (props) => (
+        <RealDispatchWrapper
+          initialState={stateFactory.build({ searchPageState: activeSearch })}
+          {...props}
+        />
```

Comment:
thought(non-blocking): what do you think of deduplicating function passed to the the wrapper in render by either wraping or changing `RealDispatchWrapper` to be `(initialState) => (props) => ReactElement`?

Reply:
I personally like that we get to continue to write JSX with the form you have now. But having to duplicate `(props) {...props}` seems like it could be pulled out

Reply:
Yeah I see what you mean! I'm thinking of switching this back to the form <Wrapper><ElementUnderTest/><Wrapper> instead of using the `wrapper` render option -- the `props` here is only passing the child element (`ElementUnderTest`). 

I also personally like getting to write/read these tests as JSX, and this seems like an easy change to reduce that duplication. Any opposition?

Reply:
Works for me 👍🏻